### DISCUSSION TITLE
Hide version

### DISCUSSION THREAD

**Discussion by ShlomiPorush** [tomasrudh/analogclock#6] [Q&A] [answered]
Hi 

Can I hide the version number? 
thanks!

**Comment by ShlomiPorush** [comment, marked as answer]
Sorry. it was WeekNumber and can be hidden by "hide_WeekNumber: true"

**Comment by tomasrudh** [comment]
Hi, yes, the number to the left is the week number, I am not sure how to mark it to be clearer. Maybe it should be set as disabled by default.

Thanks for contacting me though.
